index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Parks</title>
+        <header>
+            <img src="https://i.pinimg.com/736x/e0/b3/56/e0b35696d26ad189cb3301364c1bada1.jpg" alt="Header Image">
+            <h1>Cider Falls</h1>
+        </header>
+        <link rel="stylesheet" href="./styles/main.css">
+    </head>
+
+    <body>
+        <main id = "container">
+            //Where the DOM variable from main.js will go
+
+        </main>
+
+        <script type="module" src="./scripts/main.js"></script>
+    </body>
+
+    <footer>
+        <p>Email Address: -------------------------</p>
+        <p>Street Address: ----------------------</p>
+        <p>Phone Number: ----------</p>
+    </footer>
```

Comment:
Same as header, footer element should be right above script tags inside body tag

Reply:
It should be updated now